test/org/freedesktop/gstreamer/PadProbeTypeTest.java

```diff
@@ -0,0 +1,51 @@
+/* 
+ * Copyright (c) 2020 Neil C Smith
+ * 
+ * This file is part of gstreamer-java.
+ *
+ * This code is free software: you can redistribute it and/or modify it under 
+ * the terms of the GNU Lesser General Public License version 3 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT 
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or 
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License 
+ * version 3 for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * version 3 along with this work.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.freedesktop.gstreamer;
+
+import java.lang.reflect.Field;
+import java.util.Set;
+import org.freedesktop.gstreamer.glib.NativeFlags;
+import org.freedesktop.gstreamer.lowlevel.GstPadAPI;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ *
+ */
+public class PadProbeTypeTest {
+    
+
+    @Test
+    public void testCombinations() throws Exception {
+        for (Field field : PadProbeType.class.getFields()) {
+            if (Set.class.isAssignableFrom(field.getType())) {
+                Set<PadProbeType> flags = (Set<PadProbeType>) field.get(null);
+//                System.out.println(field.getName() + " : " + flags);
```

Comment:
Consider using a logger here or remove it?

Reply:
Right now, no, but in general we need to refactor a bunch of stuff in tests.  There's a whole mix of stuff that's output to System.out, some commented out, some not.  It's useful stuff at times locally.  Ideally will refactor the lot and put behind a flag / through logging.

Reply:
Need tests on Query